Makefile.toml

```diff
@@ -11,10 +9,7 @@ args = [
   "--bins",
   "--tests",
   "--examples",
-  "--release",
```

Comment:
not sure whether it is good to remove the `--release`.   Feel free to close this if that is ok.

Reply:
Good spotting.  In this case, it's what we want to do:

`--release` disables debug assertions.  Recently we had a problem with [debug assertions being broken](https://github.com/scroll-tech/ceno/issues/715).  And we already [enabled them for integration tests](https://github.com/scroll-tech/ceno/pull/714/files).

Reply:
In compensation, we upped the default optimisation level for the dev-build to 2.  As a compromise between speed of building vs speed of execution.